Formula/libsais.rb

```diff
@@ -16,13 +16,15 @@ class Libsais < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "57ef2fd00a0946441ec53636ae21ff16296106aede9d363ec3a9823d892b3b91"
   end
 
-  def install
-    system ENV.cc, "-c", *Dir.glob("*.c")
-    system ENV.cc, "-shared", "-o", shared_library("libsais"), *Dir.glob("*.o")
-    system "ar", "rcs", "libsais.a", *Dir.glob("*.o")
+  depends_on "cmake" => :build
 
-    include.install Dir.glob("*.h")
-    lib.install shared_library("libsais"), "libsais.a"
+  def install
+    system "cmake", "-S", ".", "-B", "build",
+                    *std_cmake_args
```

Comment:
```suggestion
    system "cmake", "-S", ".", "-B", "build",
                    "-DBUILD_SHARED_LIBS=ON",
                    *std_cmake_args
```
You'll need to adjust the stuff installed below tho.

Reply:
Looks like they use `LIBSAIS_BUILD_SHARED_LIB` here.